test/cql-pytest/test_cast_data.py

```diff
@@ -100,6 +100,21 @@ def test_cast_from_large_varint_to_varchar(cql, table1, cassandra_bug):
     cql.execute(f'INSERT INTO {table1} (p, cVarint) VALUES ({p}, {v})')
     assert [(str(v),)] == list(cql.execute(f"SELECT CAST(cVarint AS varchar) FROM {table1} WHERE p={p}"))
 
+# In test_cast_column_names we checked that columns are named in Cassandra style
+def test_cast_column_names(cql, table1, cassandra_bug):    
+    p = unique_key_int()
+    v = 32767456456456456456545678943512357658768763546575675
```

Comment:
Nitpick: maybe use a small value (e.g., "7") to not mix this test with the issue of what happens when casting a huge varint to a small type? This matter has no relevance to this test.